src/2/world5.md

```diff
@@ -693,167 +693,167 @@ And then towards light magic formation...
 Angel palace.
 Highest floor - Sky hall [Azure True Heaven] _TL Note:_ `蒼き真天` another weird one, need to think of a proper name?
 
-There were infinite blue skies, without ceiling or walls.
-With only solid floor under their foot.
-Other than that it was place fully exposed to skies.
-It was on the same latitude as clouds and there was no obstacles to winds that washed away enormous clouds.
-Their hair and clothes were fluttering by the wind.
-It felt as if they were standing on top of many thousands meters tall enormous spire, or maybe they were [in the middle of skies]
+There were the infinite blue skies, without ceiling or walls. 
+Other than the solid floor under their feet it was a place fully exposed to the skies.
+
+It was on the same lattitude as the enormous clouds and there were no obstacles to the wind that washed them away. 
+Their hair and clothes were fluttering in the wind. 
+It felt as if they were standing on top of some many thousands meters tall enormous spire, or maybe they were [in the middle of skies].
 And then...
 
-"Heavens are overflown with anger."
+"The heavens are overflown with anger."
+
+An arrogant looking angel with six wings was standing there right in the center of an altar. 
+It was the awe-inspiring tall figure of a floating angel, with flax hair similar to Vicious'. 
 
-There was standing arrogant looking angel with six wings, right in the center of an altar.
-It was tall figure of floating angel.
-With similar to Vicious flax hair and awe-inspiring look.
-Amber eyes reflected strong will and wisdom, and angel's body was well built.
-There was standing foreign god's hero, wearing blue heavenly garments in addition to white one.
+His amber eyes reflected strong will and wisdom, and his body was well built. The Foreign God's hero, wearing blue heavenly garments in addition to white one, was standing there.
 
-"You who are wingless, you who crawl the ground, you who looks up at the heaven.
-Why cannot you obey?
-Like healing rain falling onto ground from the skies.
-Like apple falling down from air.
-Everything given to earth is blessing from heaven.
+"You who are wingless, you who crawl the ground, you who looks up at the heaven. 
+Why cannot you obey? 
+The healing rains falling onto the ground from the skies. 
+Apples falling down from trees. 
+Everything given to the earth is a blessing from the heavens. 
 Why cannot you comprehend this providence?"
 
-Said angel who was looking up at the skies.
+Said the angel who was looking up at the skies.
 
 "This is why <span title="Angels">we</span> are standing at the top."
 
-Behind was a cross, stuck to the floor.
-These enormous things, that could fit entire human, would be rather coffins in the shape of the cross.
-In this red semi-transparent cross...
-Inside could be seen locked up struggling figure.
```

Comment:
I suggest the following (checked in the raws, その内側に閉じこめられ、藻搔いている人影をカイは見た)
> The cross was red and semi-transparent...
> Locked up inside it could be seen a struggling figure.

Sorry I didn't cross-check this against the raws, just thought what would sound good. >.<

Reply:
I think we should leave the first sentence as it is `In red semi-transparent cross...`

But the second one is good, let's go with it 👍

Reply:
`Locked up inside could be seen a struggling figure` sounds good